packages/lib/src/toolbar/controls/Histogram/Histogram.tsx

```diff
@@ -75,7 +83,22 @@ function Histogram(props: Props) {
             invertColorMap={invertColorMap}
           />
         )}
-        <HistogramIndicators positions={indicatorPositions} height={height} />
+        <g className={styles.indicators}>
+          {onChange ? (
+            <InteractiveIndicators
+              positions={indicatorPositions}
+              size={size}
+              onChange={(min, max) =>
+                onChange(xScale.invert(min), xScale.invert(max))
```

Comment:
This allows to only work in SVG coordinates into `InteractiveIndicators`

Reply:
Clever